.github/workflows/minor-release.yaml

```diff
@@ -39,13 +39,15 @@ jobs:
         env:
           ACTIONS_ALLOW_UNSECURE_COMMANDS: true
       - name: Update release dir and push tag
+        # we use rsync rather than a cp to ensure deleted files are reflected in the updated release dir
         run: |
-          cp -R src/imports/ releases/v${{ steps.parse_current_tag.outputs.major }}
-          cp -R src/terra-core/ releases/v${{ steps.parse_current_tag.outputs.major }}
+          rsync -avu --delete "src/imports" "releases/v${{ steps.parse_current_tag.outputs.major }}"
```

Comment:
I shift this to rsync as the previous approach would not reflect deleted files properly.